src/app/public/modules/layout/layout.component.ts

```diff
@@ -90,8 +90,11 @@ export class StacheLayoutComponent implements OnInit, StacheLayout, AfterViewIni
     }
   }
 
-  public ngAfterViewInit() {
-    this.setMinHeight();
+  public ngOnChanges(): void {
+    // Reset the wrapper height whenever there are changes.
+    this.windowRef.nativeWindow.setTimeout(() => {
+      this.setMinHeight();
```

Comment:
how do we test this?

Reply:
Good question. The min-height behavior isn't tested to begin with. The way the layout component spec is setup makes it difficult to test the min-height value easily. We may need to consider a visual test for this one. What do you think?